elasticsearch/linux_distributions/opendistro-tar-build.sh

```diff
@@ -18,26 +18,35 @@ set -e
 
 REPO_ROOT=`git rev-parse --show-toplevel`
 ROOT=`dirname $(realpath $0)`; echo $ROOT; cd $ROOT
-ES_VERSION=`$REPO_ROOT/bin/version-info --es`; echo ES_VERSION: $ES_VERSION
-OD_VERSION=`$REPO_ROOT/bin/version-info --od`; echo OD_VERSION: $OD_VERSION
-IS_CUT=`$REPO_ROOT/bin/version-info --is-cut`; echo IS_CUT: $IS_CUT
-S3_BUCKET="artifacts.opendistroforelasticsearch.amazon.com"
-ARTIFACTS_URL="https://d3g5vo6xdbdb9a.cloudfront.net"
+MANIFEST_FILE=$REPO_ROOT/release-tools/scripts/manifest.yml
```

Comment:
Use $ROOT/manifest.yml, so that it will still work if the script folder changes.